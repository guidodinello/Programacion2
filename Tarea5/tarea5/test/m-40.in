# 
insertarEnColCadenas (1,0) 1
insertarEnColCadenas (1,1) 1
removerDeColCadenas 1 1
estaEnColCadenas 1 1
infoEnColCadenas 1 1
insertarEnAvl 10
arregloAAvl 5 1 2 3 4 5
enOrdenAvl
avlMin 4
avlABinario
singleton 20
unionDeConjuntos 3 10 20 30
diferenciaDeConjuntos 2 20 40
interseccionDeConjuntos 2 30 50
arregloAConjunto 5 1 2 3 4 5
Fin
