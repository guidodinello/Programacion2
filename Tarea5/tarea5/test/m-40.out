1>#  .
2>Se insertó 1 en la cadena 1.
3>Se insertó 1 en la cadena 1.
4>Se removió la primera aparición de 1 de la cadena 1.
5>1 está en la cadena 1.
6>La primera aparición de 1 en la cadena 1 es (1,0.00)
7>Insertado.
8>
--(5,0.00)
-(4,0.00)
(3,0.00)
--(2,0.00)
-(1,0.00)
9>1 2 3 4 5 
10>
-(7,0.00)
--(6,0.00)
(5,0.00)
--(4,0.00)
-(3,0.00)
--(2,0.00)
---(1,0.00)
11>
--(5,0.00)
-(4,0.00)
(3,0.00)
--(2,0.00)
-(1,0.00)
12>Creado el singleton.
13>Union.
14>Diferencia.
15>Intersección.
16>Arreglo a conjunto con 5 elementos.
17>Fin.
